typespec-tests/src/main/java/com/cadl/literalservice/models/Model.java

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.cadl.literalservice.models;
+
+import com.azure.core.annotation.Fluent;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/** The Model model. */
+@Fluent
+public final class Model {
+    /*
+     * The literal property.
+     */
+    @JsonProperty(value = "literal", required = true)
+    private String literal = "literal";
+
+    /*
+     * The optionalLiteral property.
+     */
+    @JsonProperty(value = "optionalLiteral")
+    private OptionalLiteral optionalLiteral;
+
+    /** Creates an instance of Model class. */
+    public Model() {
+        literal = "literal";
+    }
+
+    /**
+     * Get the literal property: The literal property.
+     *
+     * @return the literal value.
+     */
+    public String getLiteral() {
+        return this.literal;
+    }
```

Comment:
In client, we may need to later refine whether this should be an `Enum Literal` (instead of a String).